server/routes/yourCompletedReports/index.ts

```diff
@@ -0,0 +1,22 @@
+import express, { RequestHandler, Router } from 'express'
+import asyncMiddleware from '../../middleware/asyncMiddleware'
+
+import YourCompletedReports from './yourCompletedReports'
+
+import CompletedAdjudicationsService from '../../services/completedAdjudicationsService'
+
+export default function yourCompletedReportsRoutes({
```

Comment:
Just checking... Is the casing of the name correct?

Reply:
You mean instead of YourCompletedReportsRoutes? I think that lowercase is the norm here - or at least it seems to be with the other roots.